examples/nlp/language_modeling/conf/megatron_llama_inference.yaml

```diff
@@ -0,0 +1,37 @@
+inference:
```

Comment:
why do we need this file? shouldn't the .nemo inference be identical to any other .nemo file?

Reply:
Because llama requires `add_BOS: True` and a special EOS token `end_strings: ["</s>"]`

Reply:
I am not fully sure if it makes sense to have a new inference file for this here...

@ericharper what do you feel? we have similar issues for T5/HF T5 as well - the way we handle that currently is pretty bad, we just manually know what model file is supposed to use what inference param. But if we keep on creating a new config file for every new community model I am afraid we'll make this folder extremely messy

maybe a middle ground could be to have a new folder, maybe called `community_models` and have all these config files there? I'm still afraid we might end up having a lot of files there, but dont see any better alternative

Reply:
Ideally this info should live inside the .nemo

we have the add_bos_to_input/add_eos_to_input tags in the data config, maybe longer term we need to move that to the model config so that we can directly use it during inference

Reply:
There will be a larger effort for managing configs. Let's just make the this config make the most sense for this model for now